org.scala-ide.sdt.core.tests/src/org/scalaide/core/sbtbuilder/ScopeCompileTest.scala

```diff
@@ -61,23 +62,13 @@ object ScopeCompileTest {
   }
 
   @BeforeClass def setup(): Unit = {
-    SDTTestUtils.enableAutoBuild(false)
-    var done = false
-    List(projectAName, projectBName).foreach { name =>
-      EclipseUtils.workspaceRunnableIn(workspace) { monitor =>
-        val wspaceLoc = workspace.getRoot.getLocation
-        val src = new File(sourceWorkspaceLoc(bundleName).toFile().getAbsolutePath + File.separatorChar + name)
-        val dst = new File(wspaceLoc.toFile().getAbsolutePath + File.separatorChar + name)
-        FileUtils.copyDirectory(src, dst)
-        done = true
-      }
+    initializeProjects(bundleName, Seq(projectAName, projectBName)) {
+      projectA = createProjectInWorkspace(projectAName, withSrcOutputStructure)
+      projectB = createProjectInWorkspace(projectBName, withSrcOutputStructure)
+      addToClasspath(projectB, JavaCore.newProjectEntry(projectA.underlying.getFullPath, false))
+      projectA.underlying.refreshLocal(IResource.DEPTH_INFINITE, null)
+      projectB.underlying.refreshLocal(IResource.DEPTH_INFINITE, null)
```

Comment:
The section above looks very similar to the one above, in `@BeforeClass def setup()`. Could they be refactored? I don't mind a bit of duplication if it's clearer this way, but I'm wondering if you noticed and it was intentinoal.